tests/lazy-hclose.hs

```diff
@@ -1,65 +0,0 @@
-import qualified Data.ByteString       as S
-import qualified Data.ByteString.Char8 as S8
-
-import qualified Data.ByteString.Lazy       as L
-import qualified Data.ByteString.Lazy.Char8 as L8
-
-import Control.Monad
-import System.Directory
-import System.Mem
-import System.IO
-
-import Data.ByteString.Internal
-import Foreign.ForeignPtr
-
-main = do
```

Comment:
This one looks interesting. Could you please keep it, and ideally incorporate as an additional test suite?